docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -2196,111 +2196,156 @@
+                  "allowedValues": [
+                    "DeployIfNotExists",
+                    "Disabled"
+                  ],
+                  "metadata": {
+                    "displayName": "Effect",
+                    "description": "Enable or disable the execution of the policy"
+                  }
+                }
               },
-              "pricingTierKeyVaults": {
-                "type": "String",
-                "metadata": {
-                  "displayName": "pricingTierKeyVaults"
-                },
-                "allowedValues": [
-                  "Standard",
-                  "Free"
-                ],
-                "defaultValue": "Standard"
+              "metadata": {
+                "version": "1.0.0",
+                "category": "Security Center"
               },
-              "pricingTierKubernetesService": {
-                "type": "String",
-                "metadata": {
-                  "displayName": "pricingTierKubernetesService"
+              "policyRule": {
+                "if": {
+                  "allOf": [
+                    {
+                      "field": "type",
+                      "equals": "Microsoft.Resources/subscriptions"
+                    }
+                  ]
                 },
-                "allowedValues": [
-                  "Standard",
-                  "Free"
-                ],
-                "defaultValue": "Standard"
-              },
-              "pricingTierDns": {
-                "type": "String",
+                "then": {
+                  "effect": "[[parameters('effect')]",
+                  "details": {
+                    "type": "Microsoft.Security/securityContacts",
+                    "deploymentScope": "Subscription",
+                    "existenceScope": "Subscription",
+                    "roleDefinitionIds": [
+                      "/providers/Microsoft.Authorization/roleDefinitions/fb1c8493-542b-48eb-b624-b4c8fea62acd"
+                    ],
+                    "existenceCondition": {
+                      "allOf": [
+                        {
+                          "field": "Microsoft.Security/securityContacts/email",
+                          "contains": "[[parameters('emailSecurityContact')]"
+                        },
+                        {
+                          "field": "type",
+                          "equals": "Microsoft.Security/securityContacts"
+                        },
+                        {
+                          "field": "Microsoft.Security/securityContacts/alertNotifications",
+                          "equals": "On"
+                        },
+                        {
+                          "field": "Microsoft.Security/securityContacts/alertsToAdmins",
+                          "equals": "On"
+                        }
+                      ]
+                    },
+                    "deployment": {
+                      "location": "westeurope",
```

Comment:
I think we default to northeurope for the other policies, so should be same if possible

Reply:
done